### PR TITLE
Provide correct exception message

### LINKED ISSUE #579: MessageReceiver.CompleteAsync with empty list throws ArgumentNullException
## Actual Behavior
1. CompleteAsync(new List&lt;string&gt;())
2. throws ArgumentException with "Value cannot be null" ParamName=lockTokens

This is misleading because the parameter is NOT null. Why is it an error to supply an empty list? Just return without doing anything.

## Expected Behavior
1. CompleteAsync(new List&lt;string&gt;())
2. No exception thrown

## Versions  
- OS platform and version: Windows
- .NET Version: Standard 2.0
- NuGet package version: 3.1.0

### PR DESCRIPTION
**Pull request by SeanFeldman**
Fixes #579 
For an empty list do not throw argument null exception

### CODE REVIEW

File: src/Microsoft.Azure.ServiceBus/Resources.resx [hunk 1]
```diff
@@ -159,6 +159,9 @@
   <data name="TokenMissingExpiresOn" xml:space="preserve">
     <value>The provided token does not specify the 'ExpiresOn' value.</value>
   </data>
+  <data name="ListOfLockTokensCannotBeEmty" xml:space="preserve">
```
Comment (axisc) [inline src/Microsoft.Azure.ServiceBus/Resources.resx]: nit - please fix typo ListOfLockTokensCannotBeEmpty in the next commit

Comment (SeanFeldman) [reply src/Microsoft.Azure.ServiceBus/Resources.resx]: @axisc I'm blind as  a 🦇 - what typo?

Comment (stjepan) [reply src/Microsoft.Azure.ServiceBus/Resources.resx]: ListOfLockTokensCannotBeEmty > ListOfLockTokensCannotBeEmpty

Comment (SeanFeldman) [reply src/Microsoft.Azure.ServiceBus/Resources.resx]: Thanks @stjepan. Blind like a bat I am 😂

Comment (SeanFeldman) [reply src/Microsoft.Azure.ServiceBus/Resources.resx]: PR #583 raised

**Pull request merged**